sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/ChatUserEvent.java

```diff
@@ -21,6 +21,12 @@
      * The Id of the event recipient. A string property in notification payload.
```

Comment:
They are actually refer to the same ID.
In old Event payload, there's a property named as "recipientId" with a value of MRI without 8: prefix, which we identified as a BUG. To prevent break existing code, we added a property "recipientMri" for full MRI. 
So, here we did something special for "recipientId" & "recipientMri". In line 73 - 81 of this file, if recipientMri exists, we take that to generate the communicationIdentifier, otherwise recipientId.